README.md

```diff
@@ -1,12 +1,87 @@
-SQL Server Extension for Azure Functions
-===
+# SQL Server Extension for Azure Functions #
+
+## Introduction ##
+
+This repository contains extension code for SQL Server input and output bindings as well as a getting started guide and samples of how to use them. The getting started guide details how to setup your environment and provides a basic tutorial. A high level explanation of each binding is provided below. More in depth descriptions of each binding are in their respective sample sections. The trigger binding is currently not complete. However, if you would like to try it out and learn more about its current state, feel free to do so in the triggerREADME.
```

Comment:
Why are we saying that the trigger binding is incomplete?

Reply:
Valid points. Thanks for bringing them up. Please let me know your thoughts on the below Chris.

- Anthony and I were hesitant on pushing it as complete while knowing that there needs to be dev work. There was also the uncertainty of whether or not going forward we would be showing both the input/output and trigger bindings for the private preview. The question now is are we comfortable calling it complete/do we need to inform people about the current state it's in? What are your thoughts on this? 

A big concern was making sure people aren't expecting something fully polished. I'm not entirely familiar with how the private previews work yet, so I was under the impression that we would still get feedback even with the notes. However, if we are looking to promote it more, I can change the tone. In terms of the specific wording of 'not complete,' I'm rereading it with fresh eyes now, and I agree it could at least be rephrased if not removed.

- Originally, it started as a single README. However, through discussions with Anthony, we decided to split it into two parts. One with the input/output bindings which seems to be almost in a state that could be shipped, and one as a sort of try this with a note that it is still being developed. Again, a large part of this was due to uncertainty about the future of the SQL bindings. Thinking back, I don't think we brought this up in the weekly sync. Although, I definitely should have for the additional perspective and will try my best to do so going forward. I do agree that it is a bit weird to have two README files though. It would probably be better to have either the single README or a /docs folder. @anthonychu thoughts?

Reply:
I don't think we need to call out how ready we think the trigger is. I agree we want people to try it. I was thinking of breaking it up into 2 because it felt like it was going to be very long with them together. We can put the on one page if we have links up top to show that there are 2 sections and give the ability for someone to go directly to each one of them.

Reply:
Ok, that makes sense. I'll make some edits to not call out the state of the trigger and consolidate to one README with links.

Reply:
Updated to one README and changed call outs to trigger. @cgillum and @anthonychu thoughts?